apm-agent-plugins/apm-cassandra/apm-cassandra4-plugin/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-cassandra</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.22.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>apm-cassandra4-plugin</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+
+    <properties>
+        <apm-agent-parent.base.dir>${project.basedir}/../../..</apm-agent-parent.base.dir>
+        <maven.compiler.target>8</maven.compiler.target>
```

Comment:
If the instrumentation class can't be loaded due to a wrong class file version, it will just be ignored. We already apply that strategy for the jdk-httpclient-plugin, for example.

Reply:
Ahh, that's right 👍